packages/entities/entities-routes/src/locales/en.json

```diff
@@ -63,6 +63,19 @@
     "success_brief": "Successfully copied to clipboard"
   },
   "form": {
+    "config_flavor": {
+      "label": "Define by"
+    },
+    "config_types": {
+      "basic": {
+        "description": "Quickly set up a route with HTTP & HTTPS protocol with the essentials: path, method, and host.",
```

Comment:
Wouldn't it be better to use `paths`, `methods`, and `hosts`?

Reply:
The copy is from the design team, and I think here they refer to the general concept, so the singular form should be fine. Plus, oonly one path and host can be added in the basic mode, so the plural form could be misleading.